### PR TITLE
Add Browxy

### PR DESCRIPTION
**Pull request by dbremmen**
The new browxy version support more languages and multiple files

### CODE REVIEW

**Comment by styfle** [comment]
Hi @dbremmen thanks for the PR!

It looks like browxy doesn't support code-completion/intellisense.
Also, the syntax highlighting looks very basic.

image

However the File IO and CLI arguments as input looks nice.

Lastly, the beta link doesn't let me publish. Can I deploy a web server that emits HTML?

**Comment by dbremmen** [comment]
Dear Steven,

Thanks a lot for taking the time to try browxy and pointing out this
important issues! Sometimes we don't receive too much feedback of our users
and we don't know which items to improve
* About syntax highlighting you are right, we are using the same editor
that cloud9 uses and discovered that we need to set the language to make it
work properly (fortunately this is an easy change and we will add it in the
next release)
* About code-completion, you are right, browxy does not support code
completion now. We discussed about adding it but didn't come up with a good
solution due that each language will need a special agent that will solve
the code completion requests and maybe at this point this will add more
load to our servers (we will discuss again if we can add this feature due
that we want to have it in the medium term)
* About the publish feature is only available when the user registers (this
is because we save the program in a permanent storage associated with the
user. It is right that that is not clear in the UI (we will fix that)
* About deploying a web server that emits HTML, we didn't expose yet an
url/ip to access the running program directly, we will add it the medium
term. But even if we add that feature, the programs created with browxy are
killed after 60 seconds for resource reasons, we will increase that time
window in the future.

Due to this limitations that you've pointed out I think that browxy maybe
is in between of an IDE and Snippet runner. What we are trying to do with
browxy is to create a global repository of code ready to be run with a user
created UI. I mean for example you need a program that do some conversion
(for example URL encoder/Decoder), you access browxy, search the catalog
(we are working on enhancing the published code area) and find the program
you want, then you can execute it using the UI. If you want to add a new
feature and the user allowed viewing the source you can edit the program,
add the new feature and publish a new version of the program.
In this case the programs are don't need to execute long time, because
there are meant to display a result once for execution.

Thanks a lot for reviewing browxy again, we will fix the syntax
highlighting soon and think about the other points :D

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
 - [Coder](https://coder.com) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run most languages on a full Linux VM complete with a terminal, go-to-definition, linting, live collaboration, and auto-scaling.
 - [Gitpod](https://gitpod.io) [<img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" />](https://www.gitpod.io/gitpod-business.html) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run any language on a full Linux VM complete with terminals, GitHub and Git integration, content assist, go-to-definition, linting, live collaboration, custom Docker workspaces, and integrated code review support.
+- [Browxy.com](https://www.beta.browxy.com) - Run Java, C, C++, C#, Python and PHP. After creating a program users can publish them to a public URL as shown in this [video](https://youtu.be/DeZgd3EjG00).
```
Comment (styfle) [inline README.md]: Change the title from `Browxy.com` to `Browxy`.

Change the description so it complies with the contributing guidelines:

> Run Java, C, C++, C#, Python, or PHP with some syntax highlighting and publish to a public URL.

**Comment by styfle** [comment]
Thanks for the detailed response. Let's leave it in the Full IDE section.

I added a comment so once that gets revised, I think we can merge 👍

**Comment by dbremmen** [comment]
Awesome! I commited to my forked branch with the changes proposed in the comment. This is my first pull request update in github, I don't know if I need to do something else.
Thanks Again!

**Comment by styfle** [comment]
Looks good, thanks! 👍

**Pull request merged**